app/Loan.php

```diff
@@ -40,4 +41,9 @@ public function getCurrencyAttribute()
     {
         return $this->wallet->currency;
     }
+
+    public function setPayoffAtAttribute($value)
+    {
+        $this->attributes['payoff_at'] = Carbon::parse($value);
+    }
```

Comment:
use model $dates property to cast payoff_at to Carbon automatically

Reply:
payoff_at attribute already in the $dates property